go/libraries/doltcore/table/typed/json/writer.go

```diff
@@ -44,19 +41,7 @@ type JSONWriter struct {
 	rowsWritten int
 }
 
-func OpenJSONWriter(path string, fs filesys.WritableFS, outSch schema.Schema) (*JSONWriter, error) {
-	err := fs.MkDirs(filepath.Dir(path))
-
-	if err != nil {
-		return nil, err
-	}
-
-	wr, err := fs.OpenForWrite(path, os.ModePerm)
-
-	if err != nil {
-		return nil, err
-	}
-
+func OpenJSONWriter(wr io.WriteCloser, outSch schema.Schema) (*JSONWriter, error) {
```

Comment:
This is now just a wrapper for NewJSONWriter, so use that instead everywhere this is called